roles/rollback/tasks/main.yml

```diff
@@ -2,14 +2,45 @@
 
 - name: remove current path
   file:
-    dest={{ current_path }}
-    state=absent
+    dest: "{{ current_path }}"
+    state: absent
 
 - name: copy rollback repo to current path
-  command: cp -r {{ rollback_path }} {{ current_path }} creates={{ current_path }}
+  command: cp -r {{ rollback_path }} {{ current_path }}
+  args:
+    creates: "{{ current_path }}"
 
-- name: copy rollback db to live db
-  shell: gunzip -c {{ rollback_sql_path }} | psql -h {{ db_host }} -U {{ db_user }} {{ db }}
+- name: close active db connections
+  shell: psql -c "REVOKE CONNECT ON DATABASE {{ db }} FROM public; SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname='{{ db }}' AND pid <> pg_backend_pid();"
+  become: true
+  become_user: postgres
+
+- name: drop database
+  postgresql_db:
+    name: "{{ db }}"
+    state: absent
+  become: true
+  become_user: postgres
+
+- name: recreate database
+  postgresql_db:
+    name: "{{ db }}"
+    owner: "{{ db_user }}"
+    encoding: "UTF-8"
+    lc_ctype: "en_US.utf8"
+    lc_collate: "en_US.utf8"
+    state: present
+  become: true
+  become_user: postgres
```

Comment:
Might be worth checking whether our instances use these default `lc_ctype` and `lc_collate` locales or the corresponding local ones such as `es_ES.utf8`.

Reply:
We force the use of `en_US.utf8` here: https://github.com/openfoodfoundation/ofn-install/blob/98d35374ab99620ef95cc9cec15e470a65156219/roles/dbserver/files/fix_pg_encoding.sh

Reply:
even with that, I wouldn't give it for granted because I don't really know if we created the DBs with `bundle exec rake db:create`. Could that be forcing diferent values for these two config settings?

Reply:
I think our existing hack overrides the default template, so any databases created with `db:create` will pick up those settings from `template0`.

Reply:
I refactored this so the value is stored in a global variable `{{ postgres_encoding }}`, to make it a bit cleaner and so we could potentially override it in the future for different instances.

I also checked around various production databases to check the encoding was the same everywhere.